amundsen_application/static/js/components/ColumnList/index.tsx

```diff
@@ -153,17 +197,26 @@ const ColumnList: React.FC<ColumnListProps> = ({
         name: item.name,
         database,
       },
-      usage: hasItemStats ? item.stats[0].stat_val : '',
+      sort_order: item.sort_order,
+      usage: hasItemStats ? +item.stats[0].stat_val : null,
```

Comment:
I don't think the api guarantees the column usage will be the first statistic, so perhaps we should have a helper method to intentionally search for the `column_usage` statistic and extract the value.

Reply:
Good call!
Right now this will work, but as we add more stats it will definitely break.
I will add something now.

Reply:
+1, currently we have single stat at Lyft, but we will ingest more stat later. I think we should either define the query / metric name in config for sorting or redefine the usage not part of stat. I wonder whether we could choose the former?